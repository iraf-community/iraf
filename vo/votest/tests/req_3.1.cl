#
#  Query Registry with constraints.

# Set the test description string.
votest.descr = "Query Registry with constraints."

print ("------------------------------------------------------------------")
print ("Req  3.1:  Users shall be able to constrain the results of a search")
print ("           by supplying additional parameters to a query (e.g. ")
print ("	   bandpass, service type, etc).")
print ("------------------------------------------------------------------")


# Execute the test commands.
registry ("chandra", type="image", bandpass="x-ray", interactive=no)
