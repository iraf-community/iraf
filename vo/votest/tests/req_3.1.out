------------------------------------------------------------------
Req  3.1:  Users shall be able to constrain the results of a search
           by supplying additional parameters to a query (e.g. 
	   bandpass, service type, etc).
------------------------------------------------------------------
#
#  Found 2 records
#
#  Registry Query Constraints:
#	  Query Term: 'chandra'
#	 ServiceType:  image
#	    Bandpass:  x-ray
#
#No  ShortName          ServiceType  Identifier
#--  ---------          -----------  ----------
#
  1  Chandra            SimpleImag   ivo://cxc.harvard.edu/cda#1
  2  TGCat SIA          SimpleImag   ivo://cxc.mit/tgcat/SIA#1
